Submission.java

```diff
@@ -974,4 +976,75 @@ private void addSoundCategoryMap()
         categoryMap.put("setTempoTo:", "sound");
         categoryMap.put("tempo", "sound");
     }
+
+    /**
+     * Get the variables array.
+     *
+     * @return array of variables
+     */
+    public String[] getGlobalVariables()
+    {
+        return this.globalVariables;
```

Comment:
'this' keyword is not necessary here - since we're only referring to a single globalVariables field, just 'globalVariables' is sufficient.